pkg/lakom/resolvetag/resolvetag_suite_test.go

```diff
@@ -5,13 +5,207 @@
 package resolvetag_test
 
 import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"log"
+	"net/http/httptest"
+	"net/url"
+	"os"
 	"testing"
 
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/registry"
+	registryv1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/random"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/sigstore/cosign/v2/pkg/cosign"
+	"github.com/sigstore/cosign/v2/pkg/oci/mutate"
+	ociRemote "github.com/sigstore/cosign/v2/pkg/oci/remote"
+	"github.com/sigstore/cosign/v2/pkg/oci/signed"
+	"github.com/sigstore/cosign/v2/pkg/oci/static"
+	"github.com/sigstore/sigstore/pkg/signature"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	// Fake registry properties
+	registryURL *url.URL
+	server      *httptest.Server
+
+	signedImageFullRef   string
+	unsignedImageFullRef string
+
+	signedImageTagRef      string
+	unsignedImageTagRef    string
+	nonExistantImageTagRef string
+
+	// Key for signing images in fake registry
+	privateKey *ecdsa.PrivateKey
```

Comment:
`privateKey` also can be moved out from the global scope, it can be defined in the `BeforeSuite` and passed as parameter to `signImage`.